packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganization.tsx

```diff
@@ -38,7 +38,7 @@ export const CreateOrganization = withOrganizationsEnabledGuard(
   { mode: 'redirect' },
 );
 
-export const CreateOrganizationModal = (props: CreateOrganizationProps): JSX.Element => {
+export const CreateOrganizationModal = (props: Omit<CreateOrganizationProps, 'path'>): JSX.Element => {
```

Comment:
Shouldn't we also omit the `routing` prop here? (and for the other modals as well)

Reply:
Yeah you are tight we can also omit routing here as it's always `virtual`.